src/lib/api.js

```diff
@@ -12,7 +26,9 @@ export function fetchCharacters() {
       data.results.forEach((characters) => {
         const section = document.createElement("section");
```

Comment:
you can also put the creation of the character card in its own function (potentially in a new file -> import) and call it here